idl/KeylessShape.idl

```diff
@@ -0,0 +1,12 @@
+module shapes_demo_typesupport
```

Comment:
The issue I see with the duplicated file is the maintainability. Having the same file, if in the future the subproject is moved, the build process would fail and it will be fixed in the same PR that moves the subproject. Otherwise, modifying one file in one PR does not ensure that the other file is going to be modified accordingly and the types regenerated.